src/command/commands.rs

```diff
@@ -54,7 +54,7 @@ impl Command for Read<'_> {
 }
 
 /// Write data to a placeholder file.
-#[derive(Debug, Clone, Default)]
```

Comment:
Why did you remove Clone/Default derives from all commands?

Reply:
I think commands should not be cloned, and parameters must be specified explicitly.